timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/tripleprocessor/CollectionMembershipTripleProcessor.java

```diff
@@ -16,6 +16,7 @@ public void process(String vreName, boolean isAssertion, Triple triple) {
 
     if (isAssertion) {
       entity.addToCollection(database.findOrCreateCollection(vreName, triple.getObject()));
+      entity.removeFromCollection(database.getDefaultCollection(vreName));
```

Comment:
As discussed in the chat we should log an error and not add to the new collection if the entity already has a collection.